python-frontend/src/test/java/org/sonar/python/regex/PythonAnalyzerRegexSourceTest.java

```diff
@@ -82,7 +84,20 @@ public void test_triple_quote_literal() {
   }
 
   @Test
-  public void multiline_string_literal() {
+  public void possessiveQuantifiers() {
```

Comment:
I think it would also be nice to have a test for atomic groups (could be done as part of the other ticket, or as part of this one)